packages/core/src/renderer/components/tree-view/tree-view.tsx

```diff
@@ -47,33 +47,41 @@ export interface TreeItemProps {
 }
 
 export function TreeItem(props: TreeItemProps) {
+  const {
+    label,
+    "data-testid": dataTestId,
+    classes,
+    icon,
+    onClick,
+    selected = false,
+  } = props;
   const [hovering, setHovering] = useState(false);
   const optionalCssNames: Partial<Record<string, any>> = {};
 
-  if (props.classes?.selected) {
-    optionalCssNames[props.classes.selected] = props.selected ?? false;
+  if (classes?.selected) {
+    optionalCssNames[classes.selected] = selected;
   }
 
-  if (props.classes?.hover) {
-    optionalCssNames[props.classes.hover] = hovering;
+  if (classes?.hover) {
+    optionalCssNames[classes.hover] = hovering;
   }
 
   return (
     <li
-      className={cssNames(props.classes?.root, optionalCssNames, styles.treeItem, {
-        [styles.selected]: props.selected ?? false,
+      className={cssNames(classes?.root, optionalCssNames, styles.treeItem, {
+        [styles.selected]: selected,
       })}
       role="treeitem"
-      data-testid={props["data-testid"]}
-      onClick={props.onClick}
+      data-testid={dataTestId}
+      onClick={onClick}
       onMouseOver={() => setHovering(true)}
       onMouseLeave={() => setHovering(false)}
     >
-      <div className={cssNames(props.classes?.iconContainer, styles.iconContainer)}>
-        {props.icon}
+      <div className={cssNames(classes?.iconContainer, styles.iconContainer)}>
```

Comment:
If you would set up default value `classes={}` in props destruction you wouldn't need to repeat `?` everywhere to check object existence.